osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs

```diff
@@ -279,5 +279,7 @@ protected override void LoadComplete()
                 this.FadeTo(0.8f, 150).Then().FadeOut(judgement_fade_duration).Expire();
             }
         }
+
+        public override void Clear() => judgementsContainer.Clear(true);
```

Comment:
```suggestion
        public override void Clear() => judgementsContainer.Clear();
```

disposal argument not required; it is the default.